frontend/src/components/uploadedFileDisplay.module.css

```diff
@@ -0,0 +1,29 @@
+.uploadedFileContainer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 36px;
+  margin-bottom: 4px;
+  box-sizing: border-box;
+}
+
+.uploadedFile {
+  font-size: 14px;
+  color: var(--grey-900);
+  background-color: var(--grey-100);
+  padding: 8px 12px;
+  border-radius: 8px;
+  text-align: left;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: 65px;
+  border: 1px solid var(--grey-300);
```

Comment:
Maybe a nit-pick but I don't think we should have a border on this card here as per the UX specs

Reply:
Done :)